webapp/channels/src/components/team_selector_modal/team_selector_modal.tsx

```diff
@@ -190,9 +190,12 @@ export class TeamSelectorModal extends React.PureComponent<Props, State> {
                     </div>
                 </div>
                 <div className='more-modal__actions'>
-                    <div className='more-modal__actions--round'>
+                    <button
+                        className='more-modal__actions--round'
+                        aria-label={`Select team ${option.label}`}
```

Comment:
same question here about the translations

Reply:
Similar thing here except you can get the `intl` object by using `this.props.intl`